src/IsecPoint.py

```diff
@@ -0,0 +1,95 @@
+
+import numpy as np
+
+
+class IsecPoint:
+    """
+    Point as the result of intersection with correspondind coordinates on both surfaces
+    """
+    def __init__(self, surf1, iu1, iv1, uv1, boundary_flag, flag ,sum_idx, surf2, iu2, iv2, uv2,XYZ):
```

Comment:
Describe type and meaning of the parameters.
Consider passing UV points and patch coordinates  as numpy array.
surfA, iuvA, uvA, surfB, iuvB, uvB, then other flags